qdrant-landing/content/blog/qdrant-1.8.x.md

```diff
@@ -0,0 +1,130 @@
+date: 2024-02-26T11:12:00-08:00
+author: Mike Jang
+featured: false 
+tags:
+  - vector search
+  - new features
+  - sparse vector performance
+  - dymamic CPU saturation
+  - text immutability
+  - text loading optimization
+weight: 0 # Change this weight to change order of posts
+---
+
+Today, we are pleased to announce the release of [Qdrant 1.8.0](https://github.com/qdrant/qdrant/releases/tag/v1.8.0).
+We've optimized performance in a number of ways. We've addressed issues with:
+
+<!-- Shard deltas (deferred to v1.9) -->
+- Sparse vectors
+- Dynamic CPU saturation
+- Text immutability
+- Text loading optimization
+
+We've also added <!-- TBD -->.
+
+Did we miss something? We rely on your feedback to drive our development. We
+welcome your contributions, especially in our [Discord community](https://qdrant.to/discord). Join us, introduce yourself, and help us build the best vector search engine!
+
+## Improved sparse vector performance
+
+The primary focus of Qdrant is performance. It's why we build our work in Rust.
+We also want to optimze performance in "real-world" situations.
+
+We used the [NeurIPS 2023 datasets](https://big-ann-benchmarks.com/neurips23.html). We've optimized our search to improve throughput by a factor of 16.
+
+We set up test benchmarks in our [Sparse vectors benchmark](https://github.com/qdrant/sparse-vectors-benchmark) repository. For our benchmarks, we use
+moderately-sized Azure instances with separate clients and servers:
+<!-- Arnaud isn't sure we should include these details. I think they're useful for our readers. (I've removed the Azure configuration bits described elsewhere.)  -->
+
+| System | vCPU | RAM (GB) |
+|--------|------|----------|
+| Server | 8    | 32       |
+| Client | 4    | 16       |
+
+We created the test collection with:
+
+- 8 Sparse NeruIPS segments
+- `index.on_disk=false`
+
+Based on those datasets, we found the following two-dimentional histogram of
+latency per query dimension count:
+
+![Histogram with increasing latency for higher query dimensions](/blog/qdrant-1.8.x/neurIPS_bench_example.png)
+
+The colors within the scatter plot show the frequency of the results. The "red"
+points show the highest frequency.
+
+While this is a two-dimensional graph, the dataset represents over 100 dimensions! 
+
+## Optimized CPU use
+
+We continue to optimize our search to minimize the load on your hardware. One
+part of that is on CPUs. On a typical system, the loads on each CPU is 
+low, which is a waste of resources.
+
+We optimize how we use CPUs. With dynamic CPU saturation, we set an 
+`optimizer_cpu_budget` to drive the number of CPU _cores_ to saturate with
+optimization tasks. Specifically, if the value is:
+
+- `0`: Qdrant keeps one or more CPU cores unallocated (default)
+- A negative number: Qdrant subtracts this from the number of available CPU cores
+- A positive number: Qdrant assigns this exact number of CPU cores to your configuration
```

Comment:
Adding these details here may be more confusing than helpful.

The general idea is that the user does not have to configure anything (but the posibility is there).

Something like this, but better worded, is probably better:

> With dynamic CPU saturation we find the right balance on your hardware to index faster while handling all search requests as normal.

Reply:
Thanks. I hope this [updated intro](https://github.com/qdrant/landing_page/pull/648/files#diff-c37269758c74910bcc8e4dae1c4ccc2fb994517b6ef01c4a5e2aa55109daaf27R59-R60) works.

Reply:
It is not that we don't impact search at all. But we try to keep search performance at a reasonable level. That's why it's a 'good balance between indexing speed and search performance'.